docs/guides/guides/project-setup.mdx

```diff
@@ -24,7 +24,7 @@ First things first. You have to create a project with your preferred IDE!!
 
 ## <span className="color-step">Step 1: Create the application</span>
 
-Create a new C# console application using dotnet _**v7.0**_
+Create a new C# console application using dotnet version specified in the `EmptyProject.csproj`, look after property group `<TargetFramework>`.
```

Comment:
Please make the following changes:

```suggestion
Create a new C# console application using dotnet version specified in the `EmptyProject.csproj`, look for the `<TargetFramework></TargetFramework>` XML tag.
```

Reply:
That makes sense, good point out XML tag also newbie friendly 🌟 I should have put that there in the beginning.

Reply:
Np at all.